src/js/components/Accordion/__tests__/__snapshots__/Accordion-test.js.snap

```diff
@@ -154,9 +154,11 @@ exports[`Accordion AccordionPanel 1`] = `
 }
 
 .c10 {
-  max-height: 0;
-  visibility: hidden;
+  -webkit-transition: max-height 200ms,opacity 200ms;
+  transition: max-height 200ms,opacity 200ms;
+  opacity: 0;
   overflow: hidden;
+  max-height: 0;
```

Comment:
what is the need for adding `opacity` and `max-height` to the snapshots?

Reply:
`max-height` is just moving in the snapshot, see previous line 157.
`opacity` is replacing `visibility`. The old code was attempting to animate `visibility` via `transition`. But, `visibility` isn't an animatable property, `opacity` is.